VictoriaAlbert/VictoriaAlbert/SearchViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SearchViewController.swift
+//  VictoriaAlbert
+//
+//  Created by Madushani Lekam Wasam Liyanage on 11/10/16.
+//  Copyright © 2016 Madushani Lekam Wasam Liyanage. All rights reserved.
+//
+
+import UIKit
+
+class SearchViewController: UIViewController, UITextFieldDelegate {
+
+    
+    
+    @IBOutlet weak var searchTextField: UITextField!
+    
+    var searchText: String?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+
+    @IBAction func searchButtonTapped(_ sender: Any) {
+        
+        searchText = searchTextField.text
+        performSegue(withIdentifier: "SearchSegueIdentifier", sender: searchText)
```

Comment:
The same idea applies here. It's good to connect the segue to the button and then pass on the searchTextField.text right in the prepare(for segue).